setup.py

```diff
@@ -53,7 +53,6 @@
             'flake8',
             'flake8-docstrings',
             'flake8-quotes',
-            'pydocstyle',
```

Comment:
Why is an unrelated dependency being dropped? This is py**doc**style, not py**code**style.

Reply:
Fixing right now.